packages/core/docs/i18n.stories.mdx

```diff
@@ -0,0 +1,42 @@
+import { Meta, Story, Preview, API } from '@storybook/addon-docs/blocks';
+import { html, LitElement } from 'lit-element';
+
+<Meta title="Foundation/Internationalization" />
+
+# Internationalization (i18n)
+
+Clarity Core provides an i18n service to change the internal component strings for various region support easily. Clarity Core i18n can be controlled in two different ways.
+
+1. Global Scope
+2. Component Scope
+
+## Global Scope
+
+Global Scope i18n uses the `I18nService` service from `@cds/core`. The service enables the ability to override the default component strings as well as read them.
+
+```typescript
+import { I18nService } from '@cds/core';
+
+// read values
+I18nService.keys;
+
+// override global value for particular component
+I18nService.localize({ modal: { closeButtonAriaLabel: 'override value' } });
+```
+
+## Component Scope
+
+Often components need to provide better context to the action they are describing for screen readers. For example, an alert that is dismissable can have a more specific close label than simply "close".
+The i18n service provides a component level API using the `cds-i18n` attribute. This attribute/property allows the string to be overridden per component instance rather than the entire application.
```

Comment:
`rather than across the entire application`

instead of 

`rather than the entire application`